src/Sql/dbo/Stored Procedures/CollectionUser_ReadByOrganizationUserIds.sql

```diff
@@ -0,0 +1,15 @@
+CREATE PROCEDURE [dbo].[CollectionUser_ReadByOrganizationUserIds]
+    @OrganizationUserIds [dbo].[GuidIdArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
+        CU.*
+    FROM
+        [dbo].[OrganizationUser] OU
+    INNER JOIN
+        [dbo].[CollectionUser] CU ON OU.[AccessAll] = 0 AND CU.[OrganizationUserId] = [OU].[Id]
```

Comment:
smallest nitpick: both `[OU]` and `OU` are being used here

also surprised me that we don't want any users that have `AccessAll`. This is just because of how the `OrganizationUserUserDetailsView` works right? We don't include Admins or Owners?

Reply:
I did copy this from the existing `OrganizationUser_ReadWithCollectionsById` sproc so I kept the `AccessAll` check from there. 

Looking at it again, I believe it's to avoid listing leftover `CollectionUser` associations for users with `AccessAll`. If they have the access to all collections, then any `CollectionUser` entities created (before they were given `AccessAll`) are no longer necessary (but they may still exist) so we filter to avoid showing specific collections. 

I imagine it's to avoid the confusion of having a user have access to all collections, but then possibly seeing an old subset of collections being listed. 

Hopefully that makes sense, I might ask product/design to see if we want to show anything specific in the members table in case they have access to all. At the moment, those users will show no collections.

Reply:
Oh, and I'll fix the `[OU]` and `OU` mismatch from the copy paste

Reply:
That makes sense, thanks for the clarification!